macosx/Torrent.h

```diff
@@ -24,7 +24,7 @@ typedef NS_ENUM(unsigned int, TorrentDeterminationType) {
                          lib:(tr_session*)lib;
 - (instancetype)initWithTorrentStruct:(tr_torrent*)torrentStruct location:(NSString*)location lib:(tr_session*)lib;
 - (instancetype)initWithMagnetAddress:(NSString*)address location:(NSString*)location lib:(tr_session*)lib;
-- (instancetype)initWithHistory:(NSDictionary*)history lib:(tr_session*)lib forcePause:(BOOL)pause;
+- (void)shouldResumeTorrent:(Torrent*)torrent WithHistory:(NSDictionary*)history forcePause:(BOOL)pause;
```

Comment:
In objc parlour, methods with names starting with 'should' return BOOL. The usual convention is to make method calls read like valid English sentences, like if [string startsWith:smth]. I would consider renaming, there are examples across the codebase for inspiration.

Reply:
sure,
I went with `setResumeStatusForTorrent`